pyproject.toml

```
@@ -1,5 +1,131 @@
+[project]
+name = "lcviz"
+description = "A Jdaviz-based lightcurve analysis and visualization tool"
+requires-python = ">=3.8"
+authors = [
+    { name = "JDADF Developers", email = "kconroy@stsci.edu" },
+]
+keywords = [
+    "astronomy",
+    "astrophysics",
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering :: Astronomy",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+dependencies = [
+    "astropy>=5.2",
+    "jdaviz",
+    "lightkurve>=2",
+]
+dynamic = [
+    "version",
+]
+
+[project.readme]
+file = "README.rst"
+content-type = "text/x-rst"
+
+[project.license]
+file = "LICENSE.rst"
+content-type = "text/plain"
+
+[project.urls]
+Homepage = "https://github.com/spacetelescope/lcviz"
+
+[project.optional-dependencies]
+test = [
+    "pytest",
+    "pytest-astropy",
+    "pytest-tornasync",
+]
+docs = [
+    "sphinx-rtd-theme",
+    "sphinx-astropy",
+]
+
 [build-system]
-requires = ["setuptools",
-            "setuptools_scm",
-            "wheel"]
+requires = [
+    "setuptools>=61.2",
+    "setuptools_scm",
+    "wheel",
+]
 build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+zip-safe = false
+include-package-data = true
+license-files = [
+    "LICENSE.rst",
+]
+
+[tool.setuptools.packages.find]
+namespaces = false
+
+[tool.setuptools.package-data]
+lcviz = [
+    "data/*",
+    "*.vue",
+    "plugins/*/*.vue",
+]
+
+[tool.setuptools_scm]
+write_to = "lcviz/version.py"
+
+[tool.pytest.ini_options]
+minversion = "5.0"
+norecursedirs = [
+    "build",
+    "docs/_build",
+]
+testpaths = [
+    "\"lcviz\"",
+    "\"docs\"",
+]
+astropy_header = true
+doctest_plus = "enabled"
+text_file_format = "rst"
+addopts = "--doctest-rst --import-mode=append"
+filterwarnings = [
+    "error",
+    "ignore:numpy\\.ndarray size changed:RuntimeWarning",
+    "ignore:zmq\\.eventloop\\.ioloop is deprecated in pyzmq:DeprecationWarning",
+    "ignore::DeprecationWarning:glue",
+    "ignore::DeprecationWarning:bqplot",
+    "ignore::DeprecationWarning:bqplot_image_gl",
+    "ignore::DeprecationWarning:bqscales",
+    "ignore::DeprecationWarning:ipykernel",
+    "ignore::DeprecationWarning:traittypes",
+    "ignore::DeprecationWarning:voila",
+]
+
+[tool.coverage.run]
+source = [
+    "lcviz",
+]
+omit = [
+    "lcviz/tests/*",
+    "lcviz/version*",
+    "*/lcviz/tests/*",
+    "*/lcviz/version*",
+]
+
+[tool.coverage.report]
+exclude_lines = [
+    "pragma: no cover",
+    "except ImportError",
+    "raise AssertionError",
+    "raise NotImplementedError",
+    "def main\\(.*\\):",
+    "pragma: py{ignore_python_version}",
+    "def _ipython_key_completions_",
+]
+
+[tool.build_sphinx]
+edit-on-github = true
+github-project = "spacetelescope/lcviz"
```

```suggestion
github_project = "spacetelescope/lcviz"
```
Actually I lied, turns out it is an underscore in `jdaviz`.